core/src/main/kotlin/org/evomaster/core/output/service/TestSuiteWriter.kt

```diff
@@ -443,6 +456,9 @@ class TestSuiteWriter {
             } else {
                 lines.add("private static String $baseUrlOfSut = \"${BlackBoxUtils.targetUrl(config, sampler)}\";")
             }
+            if (useWireMock()) {
+                lines.add("private static WireMockServer wireMockServer;")
```

Comment:
is there going to be only one single server???
what if the SUT makes two calls to two different services using different ports?
wouldn't that need 2 WM instances running on 2 different IP loopback addresses?

Reply:
could be fine to support only 1 single instance for now, but leave TODO comment, as ll need to be handled in next iterations (eg 1 of industrial SUTs uses at least 4 different services)

Reply:
Modified it to have more servers.